customer-node/service/customerService.js

```diff
@@ -0,0 +1,9 @@
+const { findCustomer, createCustomer, updateCustomerData } = require('../data_access/customerda');
+
+const getCustomer = (customerId) => findCustomer(customerId);
+
+const addCustomer = (customer) => createCustomer(customer);
+
+const updateCustomer = (customer) => updateCustomerData(customer);
```

Comment:
What happens when you try to update a customer that does not exist?

Reply:
changed from no content to 404 error